### ISSUE TITLE
Cant print QR code

### ISSUE THREAD

**Issue by caini1213** [andresperezmelo/print_bluetooth_thermal#18]

**Comment by andresperezmelo** [comment]
To print barcodes or QR codes use the package [esc_pos_utils](https://pub.dev/packages/esc_pos_utils), check that it is generating the List<int> well with the package, also check in console if there is any message.

**Comment by caini1213** [comment]
Yes, i have installed with esc_pos_utils package, but still face error.

Is there any limit to the characters?

**Comment by caini1213** [comment]
![MicrosoftTeams-image (2)](https://github.com/andresperezmelo/print_bluetooth_thermal/assets/58871500/108c8df1-76af-4389-9d85-77757aec3baf)

**Comment by andresperezmelo** [comment]
The message there is about the printer's ability to receive packets, if you send smaller prints the printer will be able to handle them.
